src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar--start-day-sun calendar--month-length-29">
+        <div class="calendar__block calendar__block--1 "></div>
+        <div class="calendar__block calendar__block--2 "></div>
```

Comment:
```suggestion
        <div class="calendar__day"></div>
        <div class="calendar__day "></div>
```
redundant modifiers
remove from all days.
.calendar__day more descriptive class name